projects/cdk/directives/font-size/font-size.directive.ts

```diff
@@ -0,0 +1,64 @@
+import {DOCUMENT, isPlatformBrowser} from '@angular/common';
+import {DestroyRef, Directive, inject, type OnInit, PLATFORM_ID} from '@angular/core';
+import type {TuiStringHandler} from '@taiga-ui/cdk/types';
+import {tuiCreateTokenFromFactory, tuiInjectElement} from '@taiga-ui/cdk/utils';
+
+const IFRAME = 'position: fixed; visibility: hidden; pointer-events: none';
+const BODY = 'height: fit-content; line-height: 1em;';
+
+export const TUI_FONT_SIZE_HANDLER = tuiCreateTokenFromFactory<TuiStringHandler<number>>(
+    () => (size) => `${Math.max(size, 16)}`,
```

Comment:
Custom handler of current font size. For example here we scale MacOS Safari default font-size of 13 up to 16 to adhere to the sane browsers/OS